src/webui/api/appcontroller.cpp

```diff
@@ -1172,6 +1181,63 @@ void AppController::getDirectoryContentAction()
     setResult(ret);
 }
 
+void AppController::cookiesAction()
+{
+    const QList<QNetworkCookie> &cookies = Net::DownloadManager::instance()->allCookies();
```

Comment:
```suggestion
    const QList<QNetworkCookie> cookies = Net::DownloadManager::instance()->allCookies();
```

Reply:
Out of curiosity, why don't we want a reference here?

Reply:
It is returning a value and not a reference.